app/components/sideBar.tsx

```diff
@@ -29,6 +44,24 @@ export const SideBar = () => {
           );
         })}
       </nav>
+
+      {isLoggedIn && (
+        <div className={style.loggedInAccountContainer}>
+          <img
+            src={datum.avatarURL}
+            alt={`${datum.nickname ?? datum.name}'s avatar`}
+            className={style.avatar}
+          />
+          <span>{datum.nickname ?? datum.name}</span>
+        </div>
+      )}
+      {!isLoggedIn && (
+        <div className={style.loginButton}>
+          <span>
+            <Link to="/login">Sign In</Link>
+          </span>
+        </div>
+      )}
```

Comment:
```suggestion
      {isLoggedIn ? (
        <div className={style.loggedInAccountContainer}>
          <img
            src={datum.avatarURL}
            alt={`${datum.nickname ?? datum.name}'s avatar`}
            className={style.avatar}
          />
          <span>{datum.nickname ?? datum.name}</span>
        </div>
      ) : (
        <div className={style.loginButton}>
          <span>
            <Link to="/login">Sign In</Link>
          </span>
        </div>
      )}
```
今後の変更で間違えそうなので, 素直に三項演算子でいいかと思います.